### ISSUE TITLE
Parsing error:  TypeError: 'NoneType' object is not subscriptable

### ISSUE THREAD

**Issue by chrisamiller** [griffithlab/pVAC-Seq#251]
When running pvacseq, I get the below error, which doesn't make a lot of sense to me.

```
Running IEDB on Allele HLA-A*01:01 and Epitope Length 11 with Method SMMPMBEC - Entries 1-42
Completed
Parsing IEDB Output for Allele HLA-A*01:01 and Epitope Length 11 - Entries 1-42
Traceback (most recent call last):
  File "/gscuser/cmiller/usr/miniconda2/envs/python3/bin/pvacseq", line 11, in <module>
    sys.exit(main())
  File "/gscuser/cmiller/usr/miniconda2/envs/python3/lib/python3.5/site-packages/pvacseq/pvacseq.py", line 84, in main
    args[0].func.main(args[1])
  File "/gscuser/cmiller/usr/miniconda2/envs/python3/lib/python3.5/site-packages/pvacseq/lib/main.py", line 282, in main
    pipeline.execute()
  File "/gscuser/cmiller/usr/miniconda2/envs/python3/lib/python3.5/site-packages/pvacseq/lib/pipeline.py", line 253, in execute
    split_parsed_output_files = self.call_iedb_and_parse_outputs(chunks)
  File "/gscuser/cmiller/usr/miniconda2/envs/python3/lib/python3.5/site-packages/pvacseq/lib/pipeline.py", line 391, in call_iedb_and_parse_outputs
    lib.parse_output.main(params)
  File "/gscuser/cmiller/usr/miniconda2/envs/python3/lib/python3.5/site-packages/pvacseq/lib/parse_output.py", line 452, in main
    iedb_results = process_input_iedb_file(args.input_iedb_files, tsv_entries, args.key_file, args.top_result_per_mutation, args.top_score_metric)
  File "/gscuser/cmiller/usr/miniconda2/envs/python3/lib/python3.5/site-packages/pvacseq/lib/parse_output.py", line 363, in process_input_iedb_file
    iedb_results              = parse_iedb_file(input_iedb_files, tsv_entries, key_file)
  File "/gscuser/cmiller/usr/miniconda2/envs/python3/lib/python3.5/site-packages/pvacseq/lib/parse_output.py", line 269, in parse_iedb_file
    return match_wildtype_and_mutant_entries(iedb_results, wt_iedb_results)
  File "/gscuser/cmiller/usr/miniconda2/envs/python3/lib/python3.5/site-packages/pvacseq/lib/parse_output.py", line 214, in match_wildtype_and_mutant_entries
    match_wildtype_and_mutant_entry_for_frameshift(result, mt_position, wt_results, previous_result)
  File "/gscuser/cmiller/usr/miniconda2/envs/python3/lib/python3.5/site-packages/pvacseq/lib/parse_output.py", line 95, in match_wildtype_and_mutant_entry_for_frameshift
    if mutation_position == 1 and int(previous_result['mutation_position']) <= 1:
TypeError: 'NoneType' object is not subscriptable
```

Command:
`pvacseq run -b 500 -c 1 -e 8,9,10,11 112200.bed.vep.vcf 112200 "HLA-A*02:01,HLA-A*01:01,HLA-B*08:01,HLA-B*27:05,HLA-C*07:01,HLA-C*01:02" {NetMHC,NetMHCpan,PickPocket,SMM,SMMPMBEC}`
(working directory /gscmnt/gc2708/info/medseq/aml_relapse/neoepitopes/variants)

Relevant file that it's trying to parse is here:
/gscmnt/gc2708/info/medseq/aml_relapse/neoepitopes/variants/112200.bed.vep.vcf.pvacseq/MHC_Class_I/tmp/112200.smmpmbec.HLA-A*01:01.11.tsv_1-42

Output is just a header line before it dies.

Any ideas would be appreciated!

**Comment by susannasiebert** [comment]
I will look into it. Can you confirm what version of pvacseq you are running (pvacseq -v)? Should be 4.0.5.

**Comment by chrisamiller** [comment]
Hrmm - I'm on 4.0.4 - must have missed a version.  Let me upgrade and try again.

**Comment by chrisamiller** [comment]
A rerun succeeded, so this appears to be fixed in 4.0.5. Thanks!
